pyproject.toml

```diff
@@ -31,6 +31,7 @@ djangorestframework = "^3.9.1"
 djangorestframework-jsonapi = "~2"
 drf-nested-routers = "^0.91.0"
 python-dateutil = "~2.7.5"
+pynamodb = "~4.2"
```

Comment:
I'm a little leery of adding a pynamodb dependency to shipchain_common. Transmission doesn't currently have pynamodb, and would be forced to add it as a dependency with this change. Maybe this mixin belongs in Profiles for now.

Reply:
That makes sense. It's not like I use this for the ThirdPartIntegrator as there is a lot in here that's unnecessary for that project.